src/test/resources/application.conf

```diff
@@ -25,7 +25,7 @@ ergo {
     minimalSuffix = 10
 
     # Is the node is doing mining
-    mining = false
+    mining = true
```

Comment:
@kushti a lot of tests were failing because they use the mining reward as Input to make transactions ...

Reply:
Ok, but then add mining = false to mainnet.conf to avoid running miner for the existing mainnet nodes (for the testnet maybe that's okay to have mining switched on by default)

Reply:
I checked that already, in `test/it` scope, all configs are explicitly set to `mining = false` already except for `devnetTemplate.conf` with `mining = true`.